keyboards/handwired/freoduo/keymaps/default/keymap.c

```diff
@@ -144,43 +140,43 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  * |      |Val Dn|Val Up|      |             |             |      |      |      |      |
  * `-----------------------------------------------------------------------------------'
  */
-[_ADJUST] =  LAYOUT(
-  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
-  QK_BOOT  , RGB_TOG, RGB_MOD, VK_TOGG, _______, _______, _______, _______, _______, _______, _______, _______,
-  _______, RGB_HUD, RGB_HUI, _______, _______, _______, _______, _______, _______, _______, _______, _______,
-  _______, RGB_SAD, RGB_SAI, _______, _______, _______, _______, _______, _______, _______, _______, _______,
-  _______, RGB_VAD, RGB_VAI, _______, _______,                   _______, _______, _______, _______, _______
-)
+[_FN] =  LAYOUT(
+    KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  \
+    KC_PSCR, _______, _______, RGB_HUD, RGB_HUI, _______, KC_MINS, _______, _______, KC_PSCR, KC_SLCK, KC_PAUS, \
+    KC_MPRV, KC_MPLY, KC_MNXT, RGB_SAD, RGB_SAI, _______, KC_PLUS, _______, _______, _______, _______, _______, \
+    KC_VOLD, KC_MUTE, KC_VOLU, RGB_VAD, RGB_VAI, _______, _______, _______, _______, _______, _______, _______, \
+    _______, _______, RGB_TOG, RGB_RMOD,RGB_MOD,                   _______, _______, _______, RGB_M_R, RGB_M_SW \
+ )
 
 };
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
   switch (keycode) {
-    case LOWER:
+    case _LOWER:
       if (record->event.pressed) {
         layer_on(_LOWER);
-        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+        update_tri_layer(_LOWER, _RAISE, _FN);
```

Comment:
You may not be able to access the _FN layer this way.  Layer processing is done from the highest numbered layer, down. _FN is set to be lower than the raise or lower layers, so it will be processed after those layers.